stories/greyVest/error.stories.js

```diff
@@ -2,20 +2,43 @@ import React from 'react'
 import { storiesOf } from '@storybook/react'
 import decorator from './decorator'
 import { Flex } from './../../src/layout/Flex'
-import { Box, ErrorList, Input } from './../../src/themes/greyVest'
+import { Box, ErrorList, ErrorBlock, Input } from './../../src/themes/greyVest'
 
 storiesOf('Components (Grey Vest)|Error', module)
   .addDecorator(decorator)
   .addWithJSX('Text', () => <ErrorList>I am an error</ErrorList>)
   .addWithJSX('Block', () => (
-    <ErrorList block>
+    <ErrorList Component={ErrorBlock}>
```

Comment:
This does seem like a less friendly API. I think at the end of the day, these are really just two different components, ErrorList and ErrorBlocks.